### PR TITLE
[plugin][actionlog][joomla] - Notice when disabled

### PR DESCRIPTION
**Pull request by alikon**
Pull Request for Issue #124 .

### Summary of Changes
Add a notice when the actionlog-joomla plugin is disabled with a link to the plugin

screenshot from 2018-08-24 19-40-59

### CODE REVIEW

**Comment by alikon** [comment]
I'm unable to replicate, :face_with_head_bandage: 
did you have the same error without this PR ?

**Comment by Sandra97** [comment]
Yes on a fresh install. 
I always have weird issues 😉
I'll test on another install to see if I have the same error.

**Comment by Sandra97** [comment]
No issue on my other install on a distant server, all works perfectly.
Should be an issue with my Wamp.

**Comment by alikon** [comment]
:sweat_smile:

File: plugins/system/actionlogs/actionlogs.php [hunk 1]
```diff
@@ -262,4 +275,32 @@ private function clearCacheGroups(array $clearGroups, array $cacheClients = arra
...
+
+		try
+		{
+			$result = (int) $db->loadResult();
+		}
+		catch (RuntimeException $e)
+		{
+			JError::raiseWarning(500, $e->getMessage());
+		}
+		return $result;
```
Comment (mbabker) [inline plugins/system/actionlogs/actionlogs.php]: `$result` will never be declared if you have an exception, this will cause a warning.

**Comment by mbabker** [comment]
Personally, I'd find a way to do this where you still have the tab on the edit profile page and show this message in the tab.  Right now, it's pretty confusing why you're getting an alert about the plugin being disabled or what you can do to address it.

**Comment by alikon** [comment]
make sense, btw a more explanatory message can help anyway
moved the message to the tab

screenshot from 2018-08-26 09-03-47

**Comment by alikon** [comment]
closed in favour of https://github.com/joomla/joomla-cms/pull/21898

**Pull request closed without merging**